test/MLOps.NET.Tests/Catalogs/LifeCycleCatalogTests.cs

```diff
@@ -91,7 +98,29 @@ public async Task CreateRunAsync_WithoutGitCommitHash_ShouldProvideEmptyGitCommi
             var runId = await sut.CreateRunAsync(Guid.NewGuid());
 
             //Assert
-            this.runRepositoryMock.Verify(x => x.CreateRunAsync(It.IsAny<Guid>(), string.Empty), Times.Once());
+            this.runRepositoryMock.Verify(x => x.CreateRunAsync(It.IsAny<Guid>(), new List<PackageDependency>(), string.Empty), Times.Once());
+        }
+
+        [TestMethod]
+        public async Task CreateRunAsync_WithDependencies_ShouldProvidePackageDependencies()
+        {
+            //Arrange
+            var dependencies = new List<PackageDependency>
+            {
+                new PackageDependency
+                {
+                    Name = "Microsoft.ML",
+                    Version = "1.5.2"
```

Comment:
Does this mean we will need to update the version every update?

Reply:
When we end up updating our ML.NET dependency yep. I couldn't find a better way to ensure that we actually read the proper dependencies than to use a real world use case for our tests. Hopefully it shouldn't be too much work, and be pretty obvious when it occurs.